### DISCUSSION TITLE
Duda sobre inputs y outputs

### DISCUSSION THREAD

**Discussion by aisleenceron** [IIC2133-PUC/2024-1#70] [Tarea 1] [answered]
Creo que hay un output que está mal pero para mostrarlo quisiera mostrar una foto de mi output, se puede?

**Comment by Dafnemami** [comment, marked as answer]
Hola! No hay problema si solo muestras tu output y nada de tu código.

**Comment by aisleenceron** [comment]
Hola, gracias por responderme, pero al final creo que me había enredado yo.

**Reply by Dafnemami** [reply]
oki, éxito con la tarea!
